internal/plugins/helm/v1/init.go

```diff
@@ -205,7 +214,7 @@ func (p *initPlugin) GetScaffolder() (scaffold.Scaffolder, error) {
 // PostScaffold will run the required actions after the default plugin scaffold
 func (p *initPlugin) PostScaffold() error {
 
-	if p.doAPIScaffold {
+	if p.doCreateAPI {
 		return p.apiPlugin.PostScaffold()
```

Comment:
If we follow up the interface we just need to call the CreateAPI plugin and we do not need to call `p.apiPlugin.GetScaffolder()` or `p.apiPlugin.PostScaffold()`

Reply:
Right, and we can make that change to get the same behavior (and is more robust); it will require a bunch of refactoring though, so maybe in a follow-up.

Reply:
It was done in #3658